### PR TITLE
Pull request template reworked

### PR DESCRIPTION
**Pull request by comradekingu**
#### What is it?
- [x] Bug fix (user facing)
- [x] Feature (user facing)
- [ ] Code base improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- record videos
- create clones
- take over the world

#### Fixes the following issue(s)

- 

#### Relies on the following changes

- 

#### Testing apk

debug.zip

#### Agreement
- [ ] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-#### Testing apk
-<!-- Ensure that you have your changes on a new branch which has a meaningful name. This name will be used as a suffix for the app ID to allow installing and testing multiple versions of NewPipe. Do NOT name your branches like "patch-0" and "feature-1". For example, if your PR implements a bug fix for comments, an appropriate branch name would be "commentfix". -->
+#### APK testing 
+<!-- Use a new, meaningfully named branch. The name is used as a suffix for the app ID to allow installing and testing multiple versions of NewPipe. (No names like "patch-0" and "feature-1".) "commentfix" is good if your PR implements a bugfix for comments. -->
 debug.zip
 
-#### Agreement
-- [ ] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
+#### Due dilligence
+- [ ] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).
```
Comment (TobiGr) [inline .github/PULL_REQUEST_TEMPLATE.md:28]: Why did you remove the last part?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
 #### Description of the changes in your PR
-<!-- While bullet points are the norm in this section, feel free to write a text instead if you can't fit it in a list -->
+<!-- While bullet points are the norm in this section, feel free to write a text instead of a list -->
 - record videos
 - create clones
 - take over the world
 
 #### Fixes the following issue(s)
-<!-- Also add reddit or other links which are relevant to your change. -->
+<!-- Also add non-reddit or other links relevant to your change. -->
```
Comment (TobiGr) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: Why non-reddit? reddit is explicitly allowed.

**Review by TobiGr — COMMENTED** [review]
Thank you!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (comradekingu) [reply .github/PULL_REQUEST_TEMPLATE.md:28]: There isn't any agreement needed. This is a scourge, and should be avoided.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (comradekingu) [reply .github/PULL_REQUEST_TEMPLATE.md:16]: That would be Reddit. I mean "reddit" as in the verb, meaning questionable content.

Comment (comradekingu) [inline .github/PULL_REQUEST_TEMPLATE.md:16]:
```suggestion
<!-- Also add any other links relevant to your change. -->
```

Comment (TobiGr) [reply .github/PULL_REQUEST_TEMPLATE.md]: Ah. ok :D We meant Reddit.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
-- [ ] Bug fix (user facing)
+- [ ] Bugfix (user facing)
 - [ ] Feature (user facing)
-- [ ] Code base improvement (dev facing)
+- [ ] Codebase improvement (dev facing)
 - [ ] Meta improvement to the project (dev facing)
 
 #### Description of the changes in your PR
-<!-- While bullet points are the norm in this section, feel free to write a text instead if you can't fit it in a list -->
+<!-- While bullet points are the norm in this section, feel free to write a text instead of a list -->
```
Comment (opusforlife2) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
<!-- While bullet points are the norm in this section, feel free to use sentences -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
 - 
 
 #### Relies on the following changes
 <!-- Delete this if it doesn't apply to you. -->
 - 
 
-#### Testing apk
-<!-- Ensure that you have your changes on a new branch which has a meaningful name. This name will be used as a suffix for the app ID to allow installing and testing multiple versions of NewPipe. Do NOT name your branches like "patch-0" and "feature-1". For example, if your PR implements a bug fix for comments, an appropriate branch name would be "commentfix". -->
+#### APK testing 
+<!-- Use a new, meaningfully named branch. The name is used as a suffix for the app ID to allow installing and testing multiple versions of NewPipe. (No names like "patch-0" and "feature-1".) "commentfix" is good if your PR implements a bugfix for comments. -->
```
Comment (opusforlife2) [inline .github/PULL_REQUEST_TEMPLATE.md:24]:
```suggestion
<!-- Use a new, meaningfully named branch. The name is used as a suffix for the app ID to allow installing and testing multiple versions of NewPipe, e.g. "commentfix", if your PR implements a bugfix for comments. (No names like "patch-0" and "feature-1".)  -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,28 +1,28 @@
...
 debug.zip
 
-#### Agreement
-- [ ] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
+#### Due dilligence
```
Comment (opusforlife2) [inline .github/PULL_REQUEST_TEMPLATE.md:27]:
```suggestion
#### Due diligence
```

**Review by opusforlife2 — COMMENTED** [review]
Small grammar stuff.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (comradekingu) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
<!-- While bullet points are the norm in this section, feel free to write free-form text instead of a list -->
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (comradekingu) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes, but saying it like that would be entrapment. :)

Comment (opusforlife2) [reply .github/PULL_REQUEST_TEMPLATE.md]: > I mean "reddit" as in the verb, meaning questionable content.

I don't think 'reddit' is a word in English at all.

**Comment by opusforlife2** [comment]
I like the term 'due diligence'. We should have this section in the bug report template as well. A list of checkboxes asking if the user is on the latest version, has checked for duplicates, etc.

**Review by TobiGr — APPROVED** [review]
Thanks again!

**Review by opusforlife2 — APPROVED** [review]
